change/@microsoft-teams-js-51980b95-c86a-4b3c-a2d0-1b9e475014b3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Because this change is fixing a bug and making the sdk work the way we already said it did when 2.* was released, I think this change should be considered a minor change instead of a major one.